projects/components/src/open-in-new-tab/open-in-new-tab.component.ts

```diff
@@ -1,46 +1,23 @@
 import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
 import { IconType } from '@hypertrace/assets-library';
-import {
-  ExternalNavigationWindowHandling,
-  NavigationParamsType,
-  NavigationService,
-  TimeRangeService
-} from '@hypertrace/common';
-import { ButtonSize, ButtonStyle } from '../button/button';
+import { ExternalNavigationParamsNew } from '@hypertrace/common';
+import { IconSize } from '../icon/icon-size';
 
 @Component({
   selector: 'ht-open-in-new-tab',
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <div class="open-in-new-tab" htTooltip="Open in a new tab">
-      <ht-button
-        class="open-in-new-tab-button"
-        display="${ButtonStyle.Outlined}"
-        [size]="this.size"
-        icon="${IconType.OpenInNewTab}"
-        (click)="this.openInNewTab()"
-      ></ht-button>
+    <div *ngIf="this.paramsOrUrl" class="open-in-new-tab" htTooltip="Open in a new tab">
+      <ht-link [paramsOrUrl]="this.paramsOrUrl">
+        <ht-icon icon="${IconType.OpenInNewTab}" [size]="this.iconSize"></ht-icon>
+      </ht-link>
     </div>
   `
 })
 export class OpenInNewTabComponent {
   @Input()
-  public size?: ButtonSize = ButtonSize.Small;
+  public paramsOrUrl?: ExternalNavigationParamsNew | string;
```

Comment:
- Because this component is named as `OpenInNewTab`, i feel it should be able to take both`InApp` and `External` params or a url and build a local `External` Param object. This would mean that no matter what url or params you pass, this component would always open it in a new tab. I think we can build the local param inside `ngOnchanges` with reasonable defaults for missing properties.
- 
- Could we please rename `ExternalNavigationParamsNew` to `ExternalNavigationParams` ?

Reply:
thoughts? cc @aaron-steinfeld

Reply:
Agree with the rename - how did that even get in?!

Also, yes - I think ideally the open in new tab component should be able to support and convert inApp params to external ones, but I personally don't feel it needs to hold up this PR, since we can add it later with fully backwards compatible support as it's a union type already.

Reply:
I missed it when i was migrating to the new interface. But you approved the PR, so....

Reply:
Sure we can make that change separately